src/assessments/seo/FunctionWordsInKeyphraseAssessment.js

```diff
@@ -0,0 +1,83 @@
+import { escape, merge } from "lodash-es";
+
+import Assessment from "../../assessment";
+import AssessmentResult from "../../values/AssessmentResult";
+
+/**
+ * Assessment to check whether the keyphrase only contains function words.
+ */
+class FunctionWordsInKeyphraseAssessment extends Assessment {
+	/**
+	 * Sets the identifier and the config.
+	 *
+	 * @param {Object} [config] The configuration to use.
+	 * @param {number} [config.scores.onlyFunctionWords] The score to return if the keyphrase contains only function words.
+	 * @param {string} [config.urlTitle] The URL to the relevant KB article.
```

Comment:
As a note; this is really more the anchor opening tag. I get that this is the whole code base and not introduced here.